src/components/WebcamCapture/WebcamCapture.js

```diff
@@ -73,7 +108,13 @@ class WebcamCapture extends Component {
     }
 
     if (this.state.cameraConnected) {
-      return (
+      return this.state.fakeWebcam ? (
+        <input
+          type="file"
+          accept="image/*"
+          ref={fileInput => (this.fileInput = fileInput)}
```

Comment:
In newer revisions of react the convention has changed for handling refs. Can add `fileInput = React.createRef()` below line 26 then change this to `ref={this.fileInput}`. Saves the function being defined on every render.

Reply:
Switched the declatation to use the suggested.